staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go

```diff
@@ -263,10 +264,15 @@ func (f *ConfigFlags) toDiscoveryClient() (discovery.CachedDiscoveryInterface, e
 		return nil, err
 	}
 
-	// The more groups you have, the more discovery requests you need to make.
-	// given 25 groups (our groups + a few custom resources) with one-ish version each, discovery needs to make 50 requests
-	// double it just so we don't end up here again for a while.  This config is only used for discovery.
-	config.Burst = f.discoveryBurst
+	// We don't rate limit discovery by default, preferring to defer to APF.
+	// https://github.com/kubernetes/enhancements/tree/11a976c/keps/sig-api-machinery/1040-priority-and-fairness
+	config.RateLimiter = flowcontrol.NewFakeAlwaysRateLimiter()
```

Comment:
The `newFakeRateLimiter()` produces a configuration with `QPS: 1` .... was that expected knowing that a value of `-1`  disables client-side throttling and defers it to  `FPA` ?

Why was this PR closed by the way ?